### PR TITLE
Add additional pings instructions

### PR DESCRIPTION
**Pull request by ebrodymoore**
Transform.js is now on GitHub, so added the link to the file instead of Dataflow file. @dadlerj do I capture why `transform.js` is primarily used?

### CODE REVIEW

File: handbook/engineering/adding_ping_data.md [hunk 1]
```diff
@@ -27,7 +27,8 @@ You will be asked the following questions:
     - Ensure a CHANGELOG entry is added, and that the two sources of truth for ping data are updated along with your PR:
       - Pings documentation: https://docs.sourcegraph.com/admin/pings
       - The Site-admin > Pings page, e.g.: https://sourcegraph.com/site-admin/pings
-3. Determine if any transformations/ETL jobs are required, and if so, add them to the [script](https://console.cloud.google.com/storage/browser/_details/sg-analytics-data/dataflow/pipelines/udf/transform.js?project=telligentsourcegraph&authuser=0&angularJsUrl=%2Fstorage%2Fbrowser%2F_details%2Fsg-analytics-data%2Fdataflow%2Fpipelines%2Fudf%2Ftransform.js%3Fproject%3Dtelligentsourcegraph%26authuser%3D1).
+3. Determine if any transformations/ETL jobs are required, and if so, add them to the [script](https://github.com/sourcegraph/analytics/blob/master/BigQuery%20Schemas/transform.js). The script is primarily for edge cases. Primarily,  as long as zeroes or nulls are being sent back instead of `""` in the case where the data point is empty.
```
Comment (ryan-blunden) [inline handbook/engineering/adding_ping_data.md]: This seems like it could be combined into a single sentence.

**Comment by ebrodymoore** [comment]
@attfarhan  I was already making a change to the instructions so I updated it a bit more - can you review and add your changes here?

**Comment by attfarhan** [comment]
@ebrodymoore added a commit with my changes: https://github.com/sourcegraph/about/pull/664/commits/c39c189d310cfd2cd04b808c53fd7a78cf99ba73

**Comment by ebrodymoore** [comment]
thanks! @attfarhan

**Pull request merged**